GoogleCloudPlatform/nodejs-docs-samples
# samples: migrate code from googleapis/nodejs-dlp

### Migrating samples from [googleapis/nodejs-dlp](https://github.com/googleapis/nodejs-dlp/tree/main/samples) into [nodejs-docs-samples/dlp](https://github.com/GoogleCloudPlatform/nodejs-docs-samples)
---

- DLP samples (#322)
- Travis: fix failing tests + update dependencies (#335)
- Cleanup App Engine samples and re-work tests. (#354)
- Upgrade to repo tools v1.4.7 (#370)
- Upgrade repo tools and regenerate READMEs. (#384)
- Upgrade repo tools and regenerate READMEs.
- Add + run dependency updating (bash) script (#401)
- Move DLP from REST to gRPC + client libraries (#413)
- Update + autogen README (#414)
- Add DLP image redaction sample (Take 2) (#441)
- Add DLP quickstart + fix incorrect comments (#442)
- Update dependencies + fix a few tests (#448)
- Build updates. (#462)
- Update dependencies (#468)
- Add DLP samples (BigQuery, DeID, RiskAnalysis) (#474)
- Repository Migration (#2)
- Fix three-hour typo (#4)
- Update sample dependencies (#5)
- Fix docs and regenerate scaffolding. (#7)
- Test on Node 9. (#8)
- Update dependencies to enable Greenkeeper 🌴 (#13)
- Clean up DLP region tags. (#26)
- GAPIC v2 code, release 0.3.0 (#23)
- Fix sample tests and system tests (#29)
- Upgrade repo-tools and regenerate scaffolding. (#30)
- Fix invalid region code (#32)
- tests: fix risk samples tests (#33)
- chore: make samples depend on the current version (#34)
- Fix region tags (#36)
- chore: bump version to 0.4.0 (#39)
- bump version to 0.5.0 (#42)
- Update redact test images (#43)
- bump version to 0.6.0 (#46)
- chore: lock files maintenance (#47)
- chore: lock files maintenance (#50)
- test: skipping failing tests (#53)
- chore: lock files maintenance (#55)
- test: updated images for redact sample tests (#57)
- fix: update linking for samples (#66)
- chore: update sample lockfiles (#68)
- refactor: drop repo-tool as an exec wrapper (#69)
- fix(deps): update dependency yargs to v12 (#72)
- chore(deps): lock file maintenance (#73)
- Fix dlp code sample end/start tags for embedding (#76)
- chore: bump dlp to 0.7.0 (#75)
- fix: update expected images for redact samples test (#78)
- chore(deps): lock file maintenance (#82)
- chore(deps): lock file maintenance (#84)
- chore(deps): lock file maintenance (#85)
- release: nodejs-dlp v0.8.0 (#89)
- chore(deps): lock file maintenance (#90)
- chore(deps): lock file maintenance (#94)
- fix: requiring samples/ node engine >7.6 because async/await was used (#93)
- chore(deps): lock file maintenance (#99)
- feat: Add code samples for DLP text redaction (#61)
- chore(deps): lock file maintenance (#103)
- chore: ignore package-lock.json (#105)
- DLP sample: Add autoPopulateTimespan option for creating job triggers. (#64)
- Release nodejs-dlp v0.9.0 (#113)
- fix(deps): update dependency @google-cloud/pubsub to ^0.20.0 (#137)
- Enable prefer-const in the eslint config (#141)
- tests: construct pubsub properly (#143)
- chore(deps): update dependency @google-cloud/nodejs-repo-tools to v3 (#180)
- fix(deps): update dependency @google-cloud/pubsub to ^0.21.0 (#183)
- build: fix the lint rules (#187)
- docs(samples): samples to use async/await (#190)
- refactor(samples): convert sample tests from ava to mocha (#186)
- fix(deps): update dependency @google-cloud/pubsub to ^0.22.0 (#192)
- samples: Added custom dictionary and regex code samples (#204)
- Release v0.10.0 (#215)
- fix(samples-test): increase likelihood threshold (#222)
- fix(deps): update dependency @google-cloud/pubsub to ^0.23.0 (#219)
- fix(deps): update dependency @google-cloud/pubsub to ^0.24.0 (#225)
- refactor: modernize the sample tests (#237)
- fix(deps): update dependency yargs to v13 (#235)
- fix(deps): update dependency @google-cloud/pubsub to ^0.25.0 (#244)
- chore(deps): update dependency mocha to v6
- fix(test): increase minLikelihood option threshold to VERY_LIKELY (#248)
- fix(deps): update dependency @google-cloud/pubsub to ^0.27.0 (#246)
- fix(deps): update dependency @google-cloud/pubsub to ^0.28.0 (#253)
- Release v0.11.0 (#255)
- refactor: use execSync for tests (#262)
- Release v0.12.0 (#268)
- fix(deps): update dependency @google-cloud/pubsub to ^0.29.0 (#283)
- chore: release 1.0.0 (#288)
- feat: auto-generate READMEs, add .repo-metdata.json (#293)
- chore: release 1.1.0 (#294)
- chore: release 1.2.0 (#300)
- refactor: changes formatting of several statements
- docs(samples): fix misleading comment (#304)
- chore(deps): update dependency pixelmatch to v5 (#303)
- chore: release 1.2.1 (#309)
- chore: release 1.3.0 (#312)
- chore: release 1.3.1 (#314)
- chore: release 1.4.0 (#318)
- tests: sample integration tests relied on deleted dataset (#334)
- fix(deps): update dependency yargs to v14 (#328)
- chore(deps): update dependency @google-cloud/pubsub to ^0.32.0 (#327)
- chore: release 1.5.0 (#335)
- fix(deps): update dependency @google-cloud/pubsub to v1 (#339)
- chore: release 1.6.1 (#352)
- chore: release 1.7.0 (#359)
- fix(deps): update dependency yargs to v15 (#361)
- chore: release 1.7.1 (#362)
- chore: update license headers
- chore: release 1.8.0 (#366)
- refactor: use explicit mocha imports (#378)
- chore: release 1.9.0 (#379)
- chore(deps): update dependency mocha to v7 (#384)
- chore: release 2.0.0 (#394)
- chore: release 2.0.1 (#404)
- chore(deps): update dependency uuid to v7
- chore: release 2.1.0 (#411)
- chore: release 2.2.0 (#421)
- feat!: drop node8 support, support for async iterators (#440)
- fix: sample inspect customInfoTypes infoType name (#439)
- fix: remove eslint, update gax, fix generated protos, run the generator (#449)
- chore(deps): update dependency pngjs to v4 (#447)
- chore: linting
- chore(deps): update dependency pngjs to v5 (#459)
- chore(deps): update dependency uuid to v8 (#466)
- fix(deps): update dependency @google-cloud/pubsub to v2 (#475)
- chore: release 3.0.0 (#441)
- test: upstream breaking change broke tests (#484)
- chore(deps): update dependency mocha to v8 (#482)
- docs(samples): Added sample for deid with string replacement (#483)
- chore: release 3.0.1 (#487)
- fix(samples): include locations/global in parent field (#488)
- test: make redact image comparison less strict (#500)
- chore: release 3.0.2 (#490)
- chore: refactor all DLP samples and tests (#520)
- test: address nightly flake (#535)
- fix(deps): update dependency yargs to v16 (#536)
- chore: release 3.0.3 (#537)
- chore: release 3.0.4 (#549)
- docs: add pattern reognition for running samples (#552)
- chore(deps): update dependency pngjs to v6 (#556)
- chore: release 3.0.5 (#561)
- chore: release 3.0.6 (#564)
- chore: release 3.1.0 (#570)
- chore: release 3.1.1 (#607)
- chore: release 3.1.2 (#616)
- chore: release 3.1.3 (#624)
- chore: release 3.1.4 (#630)
- chore: release 3.1.5 (#633)
- chore: release 3.1.6 (#635)
- chore: release 3.1.7 (#643)
- chore: release 3.1.8 (#645)
- chore: release 3.2.0 (#647)
- docs(samples): add auto-generated Node samples (#656)
- fix(deps): update dependency mime to v3 (#663)
- chore: release 3.2.1 (#664)
- docs(samples): removed test console logs (#662)
- chore: release 3.3.0 (#668)
- docs(samples): modified region tag to align with other snippets (#687)
- chore(main): release 3.4.0 (#694)
- chore(main): release 3.5.0 (#696)
- build!: update library to use Node 12 (#713)
- chore(main): release 4.0.0 (#714)
- fix(deps): update dependency @google-cloud/pubsub to v3 (#715)
- chore(main): release 4.0.1 (#717)
- build: fix sample test (#721)
- build: increase retries for dlp test (#723)
- chore(main): release 4.0.2 (#725)
- chore(main): release 4.1.0 (#729)
- test: fix sample test for listJobs (#734)
- chore(main): release 4.1.1 (#737)
- chore(deps): update dependency uuid to v9 (#744)
- chore(main): release 4.2.0 (#745)
- chore(main): release 4.3.0 (#778)
